src/libraries/System.Composition/Directory.Build.props

```diff
@@ -1,3 +1,6 @@
 ﻿<Project>
   <Import Project="..\Directory.Build.props" />
-</Project>
\ No newline at end of file
+   <PropertyGroup>	
+    <StrongNameKeyId>Microsoft</StrongNameKeyId>	
```

Comment:
Is there an APICompat run that should have caught this or is APICompat ignoring PKT?

Reply:
That's my assumption as well. I believe APICompat currently doesn't validate the PKT in the assembly metadata:

```
.assembly extern System.Composition.AttributedModel
{
  .publickeytoken = (CC 7B 13 FF CD 2D DD 51 )                         // .{...-.Q
  .ver 5:0:0:0
}
```

Reply:
I know there were some intentional differences, back in Silverlight days and the runtime supported it.  Those should be paired with `[assembly: AssemblyFlags(AssemblyNameFlags.Retargetable)]`.  I wonder if we could add this attribute to make it compatible with folks who compiled against the bad PKT 🤔 
cc @safern who's looking at API compat scenarios.